src/WebJobs.Extensions.DurableTask/DurableTaskJobHostConfigurationExtensions.cs

```diff
@@ -40,10 +42,48 @@ public static IWebJobsBuilder AddDurableTask(this IWebJobsBuilder builder)
             serviceCollection.TryAddSingleton<IDurabilityProviderFactory, AzureStorageDurabilityProviderFactory>();
             serviceCollection.TryAddSingleton<IMessageSerializerSettingsFactory, MessageSerializerSettingsFactory>();
             serviceCollection.TryAddSingleton<IErrorSerializerSettingsFactory, ErrorSerializerSettingsFactory>();
+            serviceCollection.TryAddSingleton<IDurableClientFactory, DurableClientFactory>();
```

Comment:
I don't think we are currently using `IDurableClientFactory` in the Durable Functions extension, so we shouldn't add it to this extension method.

Reply:
@ConnorMcMahon this will allow users to start an external call, do you suggest they having to add them manually?

Reply:
The idea was to be able to start the client from anywhere including an azure durable functions, some one may need to start an orchestration in another hub or another azure function and this will be the same usage as it would be called from a console app or an asp.net core app. what do you think ?